README.adoc

```diff
@@ -203,18 +203,91 @@ execMan request: [name: 'name-of-tenant / name-of-suite'], requestType: 'request
+-- Within this checkbox key/value pair will present.
+-- Key/value is an text entry field.
 -- Key values restricted to alpha numeric and 255 characters.
--- User will enter the value.  Check for validations currently on this field.
--- Add button to add another execution parameter with key value pair.
--- User is able to delete the key value pair (EM example shows X to delete) - do the same thing.  Click on the X and the key value pair is deleted.
+-- TO enter multipple key/value pair we have provided Add button to add another key value pair.
+-- User is able to delete the key value pair by clicking on the X and the key value pair is deleted.
 -- Save button at the bottom to save the parameters.
 
+   `key`:::
+    Type: String::::
+
+    `value`:::
+    Type: String::::
+    
+image::image23.png[Execution Parameters,{thumbnail}]
+
+
+[source,json]
+
+{
+   "SuiteId":"199641ed-73dd-4eef-51bf-08dce6cbbec6",
+   "SuiteName":"Certify_Plug-In_Suite",
+   "TenantId":"83811d0b-a121-473f-742c-08dca5a0ca7b",
+   "TenantName":"smoketest_abhijeet",
+   "Processes":
+   [
+      {
+         "ProcessId":"62",
+         "ProcessPath":"",
+         "Layout":"",
+         "Recordset":"",
+         "RecordsetMode":"",
+         "MachineAttributes":"Host Name=TitanTriage03;IPv4=10.0.84.101"
+      },
+      {
+         "ProcessId":"64",
+         "ProcessPath":"",
+         "Layout":"",
+         "Recordset":"",
+         "RecordsetMode":"",
+         "MachineAttributes":"Host Name=TitanTriage03;IPv4=10.0.84.101"
+      }
+   ]
+}
+
+[source,json]
+
+{
+   "SuiteExecutionResultId":"6972891a-9326-4c50-4316-08dcf2746bb4",
+   "SuiteId":"199641ed-73dd-4eef-51bf-08dce6cbbec6",
+   "SuiteExecutionResultStatusTypeId":1,
+   "Result":null,
+   "ErrorMessage":null,
+   "CreatedDate":"2024-10-22T14:28:26.32+00:00",
+   "StartedDate":null,
+   "CompletedDate":null,
+   "AbortedDate":null,
+   "ScheduledDate":null,
+   "PendingExecutions":
+   [
+      {
+         "PendingExecutionId":"0c78b9e5-a880-4ecc-7071-08dcf27bde04",
+         "SuiteId":"199641ed-73dd-4eef-51bf-08dce6cbbec6",
+         "SuiteItemId":"02d3e9ed-6131-4881-e5db-08dcf2751097",
+         "SuiteExecutionResultId":"6972891a-9326-4c50-4316-08dcf2746bb4",
+         "CertifyProcessId":62,
+         "CertifyProcessName":"Passing\\IPass WithoutRS(1)",
+         "TestsetId":null,
+         "CreatedDate":"2024-10-22T14:28:26.33+00:00"
+      },
+      {
+         "PendingExecutionId":"0b451566-fd68-4acb-7072-08dcf27bde04",
+         "SuiteId":"199641ed-73dd-4eef-51bf-08dce6cbbec6",
+         "SuiteItemId":"1679be6b-10cc-4d4f-e5dc-08dcf2751097",
+         "SuiteExecutionResultId":"6972891a-9326-4c50-4316-08dcf2746bb4",
+         "CertifyProcessId":64,
+         "CertifyProcessName":"Passing\\IPass WithoutRS(2)",
+         "TestsetId":null,
+         "CreatedDate":"2024-10-22T14:28:26.32+00:00"
+      }
+   ],
+   "ActiveExecutions":[],
+   "CompletedExecutions":[]
+}
```

Comment:
why are you putting this json in the readme document?    That doesn't really make sense. 
FYI - you may want to add something under the changelog so that there is history such as:

== changelog
3.0.0.0     2024-Nov:  added support for the features x,y, z